src/main/java/mods/eln/Eln.java

```diff
@@ -6905,12 +6905,12 @@ private void recipeMacerator() {
         maceratorRecipes.addRecipe(new Recipe(findItemStack("Tungsten Ingot"),
             new ItemStack[]{findItemStack("Tungsten Dust", 1)}, 0.5 * f));
 
-        maceratorRecipes.addRecipe(new Recipe(new ItemStack(Blocks.cobblestone),
-            new ItemStack[]{new ItemStack(Blocks.gravel)}, 1.0 * f));
+        maceratorRecipes.addRecipe(new Recipe(new ItemStack(Blocks.cobblestone,32),
+            new ItemStack[]{new ItemStack(Blocks.gravel)}, 10.0 * f));
```

Comment:
This is a massive downgrade -- 32 cobblestone for 1 gravel. No, cobblestone is not _that_ common, and I don't usually build a cobblegen as the first thing I do.

Reply:
Reverted based on feedback.